surefire-api/src/main/java/org/apache/maven/surefire/api/stream/AbstractStreamDecoder.java

```diff
@@ -28,6 +28,7 @@
 import java.io.EOFException;
 import java.io.File;
 import java.io.IOException;
+import java.nio.Buffer;
```

Comment:
Can we declare the variables as `Buffer` in this class instead of ByteBuffer?

Reply:
No, we cannot, see https://github.com/apache/maven-surefire/pull/332#discussion_r564955600